### PR TITLE
chore(docs): installing-superset-from-scratch guide gets complete, orderly

### PR DESCRIPTION
**Pull request by AlphaJack**
Now guide can be followed directly: `venv` is the only Python environment mentioned, needed environmental variables are stored on a file and sourced from there. All `superset` commands can now be run without interruptions

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by sfirke** [comment]
This looks like a wonderful PR!  People get tripped up by these steps and the way you reordered and added necessary config variables should help a lot.  It looks good to me, but I have never actually installed Superset from scratch (I only have deployed with Docker) so don't feel qualified to merge it myself.  Hopefully someone else can take a quick peek, agrees with me, and we can merge ASAP.

**Review by sfirke — APPROVED** [review]

File: docs/docs/installation/installing-superset-from-scratch.mdx [hunk 1]
```diff
@@ -117,27 +101,34 @@ command line.
...
+:::
+
 First, start by installing `apache-superset`:
 
 ```
 pip install apache-superset
 ```
 
-Then, you need to initialize the database:
+Then, save some environmental variables needed by superset:
```
Comment (sfirke) [inline docs/docs/installation/installing-superset-from-scratch.mdx:114]:
```suggestion
Then save some environmental variables required by Superset.  Make note of this randomly-generated SUPERSET_SECRET_KEY, as without it you may be unable to decrypt the application's data in the future.
```

**Review by sfirke — COMMENTED** [review]
suggested one comment re: importance of not losing your secret key

File: docs/docs/installation/installing-superset-from-scratch.mdx [hunk 2]
```diff
@@ -87,28 +87,12 @@ These will now be available when pip installing requirements.
...
-. venv/bin/activate
-```
-
-Or with pyenv-virtualenv:
-
-```
-# Here we name the virtual env 'superset'
-pyenv virtualenv superset
-pyenv activate superset
+python3 -m venv "venv"
```
Comment (john-bodley) [inline docs/docs/installation/installing-superset-from-scratch.mdx:94]:
```suggestion
python3 -m venv venv
```

**Comment by AlphaJack** [comment]
@sfirke Based on the "Installing from Scratch" guide I created [`superset-venv`](https://aur.archlinux.org/cgit/aur.git/tree/?h=superset-venv), an AUR package for Arch Linux users.

Unfortunately it was not possible to create a non-venv [`superset`](https://aur.archlinux.org/packages/superset), as the app requires specific dependency versions, either too old or not yet available from Arch repos.

In both packages I included a [message](https://aur.archlinux.org/cgit/aur.git/tree/superset.install?h=superset-venv) with the manual steps needed to finalize the installation.

**Review by rusackas — APPROVED** [review]

**Comment by rusackas** [comment]
I think the file this was editing is no longer there, as things got shuffled around a bit. This needs a rebase, but might also need additional tweaking after doing so. I'll convert it to a draft for the moment since it's not mergeable, but I hopeful that we can merge it soon if updates are made :) Thanks for the patience... we drowned in PRs for a bit, and are just coming around to some loose ends.

**Comment by sfirke** [comment]
Doh, I didn't remember this when I made #28221 which overlaps heavily with this one.  We can probably tackle both together.

**Pull request closed without merging**